### PR TITLE
Work In Progress: django management commands

### PR DESCRIPTION
**Pull request by tingletech**
create `management/commands` 
move management commands into it

delivers #4

`./src/manage.py duplicate_last_names` creates a report of duplicate last names used for account reconcilation

`./src/manage.py move_preprints active_user proxy_user` moves imported osf.io items to an active account

TODO: 

 - [x] prospective update `proxy_account` email to the `active_account` email in the pre-print metadata as part of `move_preprints`
 - [ ] retrospective update of for accounts that have already been merged
 - [x] and a “I’m about to merge {proxy_user}’s preprints into {active_user}’s account, does this look right [y/N]” type message
 - [x] add command to run the query for the reconciliation report

### CODE REVIEW

**Comment by tingletech** [comment]
```
python ./src/manage.py move_preprints XYZ@ucop.example.edu zvdxh@eartharxiv.example.org
active_user
	None None (XYZ@ucop.example.edu)
will become the owner of preprints from the proxy user 
	Benjamin Burger (zvdxh@eartharxiv.example.org)
Are you sure? (yes/no)
 yes
✅ preprint moved
```

File: management/commands/duplicate_last_names.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   FROM janeway.core_account
+   WHERE is_active=1
+   GROUP BY last_name HAVING COUNT(*) > 1 
+)
+ORDER BY last_name;
+"""
+
+
+class Command(BaseCommand):
+    help = "product report of duplicate last names"
```
Comment (tingletech) [inline management/commands/duplicate_last_names.py]: the help text is awkward here

**Comment by tingletech** [comment]
humm, this won't actually work out super smooth -- `repository_author.email_address` needs to be unique

**Comment by tingletech** [comment]
yea, no.  Won't work if the new email address is already attached to an article.

Also, sometimes there are ORCIDs in one or the other.

**Review by hardyoyo — APPROVED** [review]
This all looks correct to me, and has been in production use for a while. 👍

**Comment by hardyoyo** [comment]
Merging as-is, more tweaks can come in as future PRs, as discussed in Slack.

**Pull request merged**